dashboard/app/views/followers/student_user_new.html.haml

```diff
@@ -4,10 +4,13 @@
 - if current_user || !@section
   %h2= t('join_section.code.title')
   %p= t('join_section.code.instructions')
+  - display_captcha = current_user && current_user.display_captcha?
   = form_tag(student_user_new_path) do
     = text_field_tag(:section_code, params[:section_code], placeholder: I18n.t('join_section.code.placeholder'))
+    - if display_captcha
+      = recaptcha_tags callback: "onCaptchaCompletion"
     %br/
-    = submit_tag t('join_section.code.submit'), class: 'btn btn-primary'
+    = submit_tag t('join_section.code.submit'), class: 'btn btn-primary', disabled: display_captcha, id: 'join_section'
```

Comment:
`id: 'join_section'` seems like it may be a common id name. Can you do a quick sanity check of this page to make sure this id isn't used anywhere else? (or if you can make it a class instead, that would be preferred - not necessary though.)